djangocms_admin_style/templatetags/admin_style_tags.py

```diff
@@ -19,6 +20,10 @@
 @register.simple_tag(takes_context=True)
 def current_site_name(context):
     request = context.get('request')
+
+    if not request:
```

Comment:
get_current_site supports getting a Site instance even if request is None (but it needs 'django.contrib.sites' in installed apps).